src/odc/loader/test_memreader.py

```diff
@@ -215,6 +216,10 @@ def test_raster_group_md() -> None:
     assert rgm.extra_coords[0] == coord
 
 
+@pytest.mark.skipif(
+    sys.version_info >= (3, 11),
+    reason="Zarr 3 does not currently support consolidated metadata",
+)
```

Comment:
should probably test actual zarr version not Python version where such zarr might be installed.